snuba/cli/confirm_load.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import click
+import json
+
+from confluent_kafka import Producer
+
+from snuba import settings
+from snuba.datasets.factory import DATASET_NAMES
+from snuba.snapshots.postgres_snapshot import PostgresSnapshot
+from snuba.stateful_consumer.control_protocol import TransactionData, SnapshotLoaded
+
+
+@click.command()
+@click.option('--control-topic', default=None,
+              help='Topic to produce messages onto.')
```

Comment:
Not sure that `None` is a good default here, especially considering this can be passed directly to `produce` which is either going to throw or have other unexpected behavior.